src/test/java/com/cloudant/tests/ViewsTest.java

```diff
@@ -104,13 +105,238 @@ public void byStartAndEndKey() {
         assertThat(foos.size(), is(2));
     }
 
+    /**
+     * Assert that passing a boolean key with value 'false'
+     * in query and queryView will produce a result list of
+     * all false docs.
+     */
+
+    @Test
+    public void byStartAndEndBooleanKey() {
+        multiValueKeyInit();
+
+        View query = db.view("keyarray_example/boolean")
+                .startKey(false)
+                .endKey(false)
+                .includeDocs(true);
```

Comment:
You're right, the includeDocs are not needed.  They were just copied from another text case.